docs/content/tutorial/step_09.ngdoc

```diff
@@ -38,16 +38,16 @@ Since we are using [Bower][bower] to install client-side dependencies, this step
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "angular": "1.5.x",
-    "angular-mocks": "1.5.x",
-    "angular-route": "1.5.x",
+    "angular": "1.6.x",
+    "angular-mocks": "1.6.x",
+    "angular-route": "1.6.x",
```

Comment:
Upgrading the tutorial form 1.5.x to 1.6.x requires more changes. There are probably more, but off the top of my head:

- The code on https://github.com/angular/angular-phonecat needs to be updated to match the changes here.
- Some `href`s (on some steps) need to take into account the new default hashPrefix (`!`).
- The code that explicitly sets the hash prefix to `!` needs to be removed (and maybe some text referring to it).

In any case, I would not make the tutorial changes as part of this PR.